server/cmd/mmctl/commands/compliance_export.go

```diff
@@ -21,17 +24,59 @@ var ComplianceExportListCmd = &cobra.Command{
 	RunE:    withClient(complianceExportListCmdF),
 }
 
+var ComplianceExportShowCmd = &cobra.Command{
+	Use:     "show [complianceExportJobID]",
+	Example: "  compliance_export show o98rj3ur83dp5dppfyk5yk6osy",
+	Short:   "Show compliance export job",
+	Args:    cobra.ExactArgs(1),
+	RunE:    withClient(complianceExportShowCmdF),
+}
+
+var ComplianceExportCancelCmd = &cobra.Command{
+	Use:     "cancel [complianceExportJobID]",
+	Example: "  compliance_export cancel o98rj3ur83dp5dppfyk5yk6osy",
+	Short:   "Cancel compliance export job",
+	Args:    cobra.ExactArgs(1),
+	RunE:    withClient(complianceExportCancelCmdF),
+}
+
 func init() {
 	ComplianceExportListCmd.Flags().Int("page", 0, "Page number to fetch for the list of compliance export jobs")
 	ComplianceExportListCmd.Flags().Int("per-page", DefaultPageSize, "Number of compliance export jobs to be fetched")
 	ComplianceExportListCmd.Flags().Bool("all", false, "Fetch all compliance export jobs. --page flag will be ignored if provided")
 
 	ComplianceExportCmd.AddCommand(
 		ComplianceExportListCmd,
+		ComplianceExportShowCmd,
+		ComplianceExportCancelCmd,
 	)
 	RootCmd.AddCommand(ComplianceExportCmd)
 }
 
 func complianceExportListCmdF(c client.Client, command *cobra.Command, args []string) error {
 	return jobListCmdF(c, command, "message_export", "")
 }
+
+func complianceExportShowCmdF(c client.Client, command *cobra.Command, args []string) error {
+	job, _, err := c.GetJob(context.TODO(), args[0])
+	if err != nil {
+		return fmt.Errorf("failed to get compliance export job: %w", err)
+	}
+
+	printJob(job)
+
+	return nil
+}
+
+func complianceExportCancelCmdF(c client.Client, command *cobra.Command, args []string) error {
+	job, _, err := c.GetJob(context.TODO(), args[0])
+	if err != nil {
+		return fmt.Errorf("failed to get compliance export job: %w", err)
+	}
+
+	if _, err := c.CancelJob(context.TODO(), job.Id); err != nil {
+		return fmt.Errorf("failed to cancel compliance export job: %w", err)
+	}
```

Comment:
Do we need to get the job first, or could we directly cancelling it? I guess getting it first is slightly better for the error?

Reply:
Right, that was the intention, but I just checked and I didn't need to do it -- the Cancel endpoint gives almost the same result, so no need for the first call. Nice catch, thanks!